soc/st/stm32/stm32u0x/power.c

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright The Zephyr Project Contributors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/init.h>
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/pm/pm.h>
+
+#include <clock_control/clock_stm32_ll_common.h>
+#include <stm32u0xx_ll_bus.h>
+#include <stm32u0xx_ll_cortex.h>
+#include <stm32u0xx_ll_pwr.h>
+#include <stm32u0xx_ll_rcc.h>
+#include <stm32u0xx_ll_system.h>
+#include <stm32u0xx_ll_utils.h>
```

Comment:
Last minute catch:
Since only LL_PWR, LL_LPM and LL_RCC are used in this file, I'm pretty sure at least 2 (bus, cortex) of those are not required (maybe 3).